test/data/mnt/boot/extlinux/extlinux.conf

```diff
@@ -2,6 +2,6 @@ DEFAULT primary
 TIMEOUT 30
 MENU TITLE Boot Options
 LABEL primary
-      MENU LABEL primary Image
-      LINUX /Image
-      APPEND ${cbootargs} ${resin_kernel_root} ro rootwait
\ No newline at end of file
+MENU LABEL primary Image
+LINUX /Image
+APPEND ${cbootargs} ${resin_kernel_root} ro rootwait isolcpus=2
```

Comment:
What are these changes?

Reply:
Any changes to the test files were by mistake. When adding types to fix the stubs some of the functions actually executed and wrote to the test files.